docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -1,6 +1,91 @@
-# Configure Clang Static Analyzer and checkers
+# Analyzer and checker configuration
 
-Checker configuration can be done through the `--saargs` analysis option which
+CodeChecker's analyzer module currently handles ClangSA and ClangTidy. The main
+purpose of this analyzer module is to hide the differences between the
+interfaces and parameterization of these two tools. Both ClangSA and ClangTidy
+have a set of checkers with fine-tuning config options and the analyzer tools
+themselves can also be configured.
+
+## Configuration of analyzer tools
+
```

Comment:
Maybe it would be good to mention the `clang-tidy:take-config-from-directory=true` option at this section too.

```
--analyzer-config [ANALYZER_CONFIG [ANALYZER_CONFIG ...]]
                        Analyzer configuration options in the following
                        format: analyzer:key=value. The collection of the
                        options can be printed with 'CodeChecker analyzers
                        --analyzer-config'. To disable the default behaviour
                        of this option you can use the 'clang-tidy:take-
                        config-from-directory=true' option. (default: ['clang-
                        tidy:HeaderFilterRegex=.*'])
```